ckanext/validation/plugin.py

```diff
@@ -252,7 +254,6 @@ def before_update(self, context, current_resource, updated_resource):
             needs_validation = True
 
         if needs_validation:
-            updated_resource['validation_status'] = 'unkown'
```

Comment:
Setting validation_status now happens using resource_patch from the resource_validation_run action.  It no longer needs to happen here.